### ISSUE TITLE
Nimdow crashes when refreshed with a bad config

### ISSUE THREAD

**Issue by PMunch** [avahe-kellenberger/nimdow#93]
**Describe the bug**
I wanted to set up shortcuts for my music player for the media keys on my keyboard. Since they wouldn't need modifier keys I defined them without a `modifier` array in the TOML file. When I refreshed nimdow it crashed and dropped me back to the shell. It would be nice if nimdow would check the config before changing and somehow show an error if the config isn't correctly formatted, similar to how i3 does it.

**To Reproduce**
1. Create an invalid config, e.g.:
``` toml
[[startProcess]]
command = "audacious --fwd"
keys = ["XF86AudioNext"]
# Notice the lack of modifiers array
```
2. Reload the config

**Expected behavior**
Some kind of error message pop-up telling me that the config was broken and that it didn't switch to it.

**Comment by avahe-kellenberger** [comment]
Good idea. I'll probably make the modifiers optional, as well

**Comment by avahe-kellenberger** [comment]
I'm now planning to make major config changes for the command line client (#72 ) which will remove `startProcess` and other settings as well.

**Comment by avahe-kellenberger** [comment]
Example:

```
09:41 <PMunch> I put this in my autostart section: "notificatcher -f /tmp/notifications \"\\e[38;2;81;159;80m{appName}:\\e[0m {hints:urgency:::\\e[38;2;231;76;60m}{summary}\\e[0m\\n\\t{body}\""
09:42 <PMunch> But I hadn't properly escaped the \e
```

**Comment by avahe-kellenberger** [comment]
For a quick and dirty fix, I'll just have errors logged out. These will be printed to stdout, and if your logging setting is true, will also be output to ~/.nimdow.log.

In the future, I may have a floating window with a list of errors.

**Comment by avahe-kellenberger** [comment]
Fixed as of v0.7.7
